docs/graphql-over-websocket.md

```diff
@@ -0,0 +1,52 @@
+# GraphQL over WebSocket
+
+The GraphQL specification doesn't dictates which transport must be used in order to execute operations. In fact, GraphQL is _transport agnostic_, so implementors can choose which protocol makes more sense for each use case.
+
+Generally, `query` and `mutation` are carried via HTTP, while `subscription` via WebSocket: this is the default behavior in `mercurius` and many other server implementations.
+
+However, you can also choose to enable `query` and `mutation` via WebSocket. If you want to do so, pass `fullWsTransport: true` on the options, like in [this example](../examples/full-ws-transport.md). You should be able to use any GraphQL client e.g. `graphql-ws`, `apollo`, `urql`, `graphql-hooks`, _etc._ in order to send every operation via WebSocket.
+
+## Protocol
+
+The GraphQL over WebSocket Protocol (i.e. the WebSocket sub-protocol) `mercurius` uses by default is [graphql-transport-ws](https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md). It also supports the protocol used by Apollo's `subscriptions-transport-ws`; however that library is **unmaintained** so it's not recommended to use.
+
+## Extensions
+
+The `extensions` field is reserved for things that implementors want to add on top of the spec.
+
+### Message structure
```

Comment:
This document does not specify enough information to implement this protocol.

Reply:
This document was already here, I just added a little bit info to it, It was added with this PR: https://github.com/mercurius-js/mercurius/pull/271

Anyways, I don't mind adding any missing stuff... the problem is, I don't understand what is missing 😅 

If you're referring to the "extensions" or "message structure" section, I haven't touched anything about it (part of the PR linked above).

If you're referring to the GraphQL over WS protocol itself, it is documented here:
> The GraphQL over WebSocket Protocol (i.e. the WebSocket sub-protocol) `mercurius` uses by default is [graphql-transport-ws](https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md).

Reply:
https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md#single-result-operation ah it's here. Let's just directly link this?

Reply:
> https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md#single-result-operation ah it's here. Let's just directly link this?

I'll add a new paragraph with that reference as well if you think it's worth! But it's just a section of the already linked protocol 😄

Reply:
Pushed some changes on the docs, I hope it's more clear now 🧹